src/lib/utils/spotify.ts

```diff
@@ -0,0 +1,223 @@
+import { openUrl } from '@tauri-apps/plugin-opener'
```

Comment:
I think I would put this in another folder maybe called integrations instead of utils. Might be worth it to put supabase.ts in there as well

Reply:
spotify might end up having its own folder inside of integrations to handle all of the other api calls you'll be making to control the player. Don't need to do that yet, but I can see that happening shortly